pkg/uiserver/uiserver.go

```diff
@@ -53,6 +61,63 @@ func RewriteAssets(fs http.FileSystem, cfg *config.Config, updater UpdateContent
 
 	rewrite("/index.html")
 	rewrite("/diagnoseReport.html")
+
+	overrideDistroAssetsRes(fs, cfg, updater)
+}
+
+func overrideDistroAssetsRes(fs http.FileSystem, cfg *config.Config, updater UpdateContentFunc) {
+	exePath, err := os.Executable()
+	if err != nil {
+		log.Fatal("Failed to get work dir", zap.Error(err))
```

Comment:
This is a executable path instead of working dir.

Reply:
Oh, yep, the error message is not correct.